### PR TITLE
Added Boolean and Double primitives. Removed unused integration testi…

### PR DESCRIPTION
**Pull request by paul-snively**
…ng sstuff from shapeless module. Added simple example-based test for shapeless module.

### CODE REVIEW

File: testing/src/test/scala/compstak/kafkastreams4s/tests/ShapelessTests.scala [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+import compstak.circe.debezium.{DebeziumFieldSchema, DebeziumSchemaPrimitive}
+import compstak.kafkastreams4s.debezium.{DebeziumCompositeType, DebeziumPrimitiveType}
+
+class ShapelessTests extends munit.FunSuite {
+  implicit val tsDP = new DebeziumPrimitiveType[Timestamp] {
+    def debeziumType: DebeziumSchemaPrimitive = DebeziumSchemaPrimitive.Int64
+  }
+
+  type CompProposal =
+    Record.`Symbol("id") -> Long, Symbol("proposalsIdx") -> Option[Long], Symbol("compBatchId") -> Option[Long], Symbol("dateCreated") -> Timestamp, Symbol("dateUpdated") -> Timestamp, Symbol("marketId") -> Option[Long], Symbol("validated") -> Boolean, Symbol("compVersionId") -> Option[Long], Symbol("verifiedBy") -> Option[String], Symbol("monthly") -> Option[Boolean], Symbol("rollbackReason") -> Option[String], Symbol("stashed") -> Boolean, Symbol("stashReason") -> Option[String]`.T
```
Comment (LukaJCB) [inline testing/src/test/scala/compstak/kafkastreams4s/tests/ShapelessTests.scala:18]: I guess there's no way to break the line here somewhere?

File: build.sbt [hunk 2]
```diff
@@ -133,14 +128,13 @@ lazy val testing = (project in file("testing"))
       "com.compstak" %% "kafka-connect-migrate" % KafkaConnectHttp4sVersion % IntegrationTest,
       "io.circe" %% "circe-literal" % CirceVersion % IntegrationTest,
       "org.http4s" %% "http4s-async-http-client" % Http4sVersion % IntegrationTest,
-      "org.tpolecat" %% "doobie-postgres" % DoobieVersion % IntegrationTest,
-      "org.scalameta" %% "munit" % MunitVersion % IntegrationTest
```
Comment (LukaJCB) [inline build.sbt:137]: Oh, we need this for the E2E test!

File: testing/src/test/scala/compstak/kafkastreams4s/tests/ShapelessTests.scala (hunk 1 above)
Comment (paul-snively) [reply testing/src/test/scala/compstak/kafkastreams4s/tests/ShapelessTests.scala:18]: Not with the embedded expression trick. I could avoid the trick, I think, if we really insisted on it.

File: build.sbt (hunk 2 above)
Comment (paul-snively) [reply build.sbt:137]: Didn't I just remove a redundant MUnit dependency? I thought there were two in that same list. :-)

**Pull request merged**